partner_resource_delivery_schedule/models/res_partner.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2023 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _name = "res.partner"
+    _inherit = ["res.partner", "resource.mixin"]
```

Comment:
Remove resource.mixing because when creating a contact, needs to have permisions for the resource manager